omc3/correction/handler.py

```diff
@@ -109,6 +119,60 @@ def correct(accel_inst: Accelerator, opt: DotDict) -> None:
     LOG.info("Finished Iterative Global Correction.")
 
 
+
+def _update_response(
+    accel_inst: Accelerator, 
+    corrected_model: pd.DataFrame,
+    corrected_elements: pd.DataFrame,
+    delta: pd.DataFrame, 
+    optics_params: Sequence[str], 
+    variable_categories: Sequence[str], 
+    update_response: bool | str,
+    ):
+    """ Create an updated response matrix.
+    
+    If we are to compute the response including the DPP, then we have to do so from MAD-X, 
+    as we do not have the analytical formulae.
+    Otherwise we go through the way of computing the response the user requested.
+
+    In case of dpp being modified in the correction, we have to add the new dpp value
+    to the accelerator instance, to run the `twiss` in madx at this dpp.
+    All other parameters are taken care of in the model/elements.
+    The dpp value has to be reset afterwards, as in the calculation of the correction,
+    the delta is added via the CHANGEPARAMTERS file.
+
+    Alternatively: We could create a copy of the accelerator instance,
+    but that would require a proper copy implementation of the class, I think (jdilly 2024)
+    """
+    # update model, not nice as "private" elements, but will be reset later
+    original_model = accel_inst.model 
```

Comment:
@JoschD Could you make sure this what you meant? I got an error that accel_inst._model did not exist. (see [most recent commit](https://github.com/pylhc/omc3/pull/466/commits/a656e3777e5f8112b9ab8de3b85b99da5166e173#diff-1022807905606346c3484a408d137bc141402d7c5f890834dedff4ac6dea5efcL144-L145))